.github/workflows/pylint.yml

```diff
@@ -0,0 +1,22 @@
+name: Pylint
+
+on: [push]
```

Comment:
branch selector missing

Reply:
Somehow I can't add your suggestion, I think it is already committed. However it doesn't show up in the commit history.

Reply:
Never mind, my bet. Added it though